docker-multinode/README.md

```diff
@@ -72,21 +75,15 @@ Once your master is up and running you can add one or more workers on different
 Clone the `kube-deploy` repo, and run [worker.sh](worker.sh) on the worker machine _with root_:
 
 ```console
-$ export MASTER_IP=<your_master_ip (e.g. 1.2.3.4)>
+$ git clone https://github.com/kubernetes/kube-deploy
 $ cd docker-multinode
+$ export MASTER_IP=${SOME_IP}
 $ ./worker.sh
 ```
 
-`Worker done!`
-
-See [here](worker.md) for detailed instructions explanation.
-
-## Deploy a DNS
-
-See [here](deployDNS.md) for instructions.
-
-## Testing your cluster
+First, the `bootstrap` docker daemon is started, then `flannel` is started as a container in the bootstrap daemon, in order to set up the overlay network.
+Then, the main docker daemon is restarted and lastly `kubelet` is launched as a container in the main docker daemon.
 
-Once your cluster has been created you can [test it out](testing.md)
+## Addons
 
-For more complete applications, please look at the [documentation](http://kubernetes.io/docs/samples/)
+kube-dns and the dashboard are deployed automatically and kube-proxy is run in a DaemonSe with `v1.3.0-alpha.5` and over
```

Comment:
DaemonSe*t* -- t missing.

I would suggest: kube-dns and dashboard are deployed automatically with `v1.3.0-alpha.5` and over

(I would drop kube-proxy handling as it is implementation detail.)